api/Navigator.json

```diff
@@ -4941,7 +4941,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "preview"
+              "version_added": "126"
```

Comment:
This update to navigator.wakeLock matches the corresponding Bugzilla issue. Related work tracking in https://github.com/mdn/content/issues/33047